ocrd_network/ocrd_network/processing_server.py

```diff
@@ -735,3 +759,21 @@ async def run_workflow(
         )
         await db_workflow_job.insert()
         return db_workflow_job.to_job_output()
+
+    async def get_workflow_info(self, workflow_job_id) -> Dict:
+        """ Return list of a workflow's processor jobs
+        """
+        try:
+            workflow_job = await db_get_workflow_job(workflow_job_id)
+        except ValueError:
+            raise HTTPException(status_code=status.HTTP_404_NOT_FOUND,
+                                detail=f"Workflow-Job with id: {workflow_job_id} not found")
+        job_ids: List[str] = [id for lst in workflow_job.processing_job_ids.values() for id in lst]
+        jobs = await db_get_processing_jobs(job_ids)
+        res = {}
+        for job in jobs:
+            res.setdefault(job.processor_name, {})
+            res[job.processor_name].setdefault(job.state.value, 0)
+            state_count = res[job.processor_name][job.state.value]
+            res[job.processor_name][job.state.value] = state_count + 1
+        return res
```

Comment:
```suggestion
        res = {}
        failed_tasks = {}
        for job in jobs:
            res.setdefault(job.processor_name, {})
            res[job.processor_name].setdefault(job.state.value, 0)
            state_count = res[job.processor_name][job.state.value]
            if job.state.value == "FAILED":
                failed_tasks[job.id] = job
            res[job.processor_name][job.state.value] = state_count + 1
        res[failed_tasks] = failed_tasks
        return res
```

Just to give you an idea - I have not tested it!

Reply:
same as above